CMakeLists.txt

```diff
@@ -103,17 +104,26 @@ include_directories( BEFORE ${PROJECT_SOURCE_DIR}/external/tk_spline )
 # add include directories
 include_directories( BEFORE
     source/control
-    source/data
     source/daq
+    source/data
     source/utility
 )
 
-
 # build this project
-add_subdirectory( source/utility )
-add_subdirectory( source/data )
-add_subdirectory( source/daq )
-add_subdirectory( source/control )
+add_subdirectory( source )
+
+if( Psyllid_ENABLE_EXECUTABLES )
```

Comment:
I don't get where this variable is set. It has to be because otherwise the applications dir should not compile?

Reply:
`${PROJECT_NAME}_ENABLE_EXECUTABLES` is set for every pbuilder project.  I really need better pbuilder documentation . . .